packages/app/src/specs/HighlightedText.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <span>
+  <span
+    :title="props.titleText"
```

Comment:
The `title` attribute isn't ideal for this, it renders differently depending on the operating system and is not exposed to keyboard users since it only appears when you hover with the mouse. I think we should be using our existing Tooltip component here. See in this screenshot, the title on the left vs the standard tooltip on the right:

<img width="1017" alt="Screen Shot 2022-06-13 at 7 32 37 AM" src="https://user-images.githubusercontent.com/8340719/173345751-9e0f18c9-9135-4387-beb9-08f7aecb1585.png">

One other issue here is that since this `HighlightedText` component is used twice to show what matches the search term, we end up having two span elements right next to each other with the same title. We only need it once. Whether we switch to tooltip or not, we can move this up a level in the DOM - to the div that wraps the two instances of `HighlightedText`? That can either have the title element or be the activator for the tooltip.

We already do the "truncate + tooltip" pattern is `HeaderBarContent.vue` which could be another good reference.

Reply:
@marktnoonan I agree that we should use tooltips here rather than the title attribute. I looked into doing that and I think that the scope is going to be more than this PR since if we go forward with tooltips I'd like to conditionally render them based on whether or not the text is actually being truncated or not, which is going to involve some extra logic.

What do you think about moving forward with this as-is and changing from title attribute to truncation in a separate task? I will move the title attribute for the spec items to a higher level in the DOM here though

Reply:
@marktnoonan I created an issue to handle this #22323

Reply:
I thought we had settled on just a delay opening (with same timing as the `title` attr), but sure let's keep the title attr for now and get this improvement in.